src/lightning/pytorch/overrides/base.py

```diff
@@ -48,58 +48,3 @@ def predict_step(self, *args: Any, **kwargs: Any) -> Any:
 
     def forward(self, *args: Any, **kwargs: Any) -> Any:
         raise NotImplementedError
-
-
-class _LightningModuleWrapperBase(_DeviceDtypeModuleMixin, torch.nn.Module):
```

Comment:
FYI, we may need to copy this class over to the external strategy implementations for compatibility